libevm/ExtVMFace.cpp

```diff
@@ -78,17 +78,20 @@ void getBalance(
 	*o_result = toEvmC(env.balance(fromEvmC(*_addr)));
 }
 
+size_t getCodeSize(evm_context* _context, evm_address const* _addr)
+{
+    auto& env = static_cast<ExtVMFace&>(*_context);
+    return env.codeSizeAt(fromEvmC(*_addr));
+}
+
 size_t getCode(byte const** o_code, evm_context* _context, evm_address const* _addr)
 {
-	auto& env = static_cast<ExtVMFace&>(*_context);
-	Address addr = fromEvmC(*_addr);
-	if (o_code != nullptr)
-	{
-		auto& code = env.codeAt(addr);
-		*o_code = code.data();
-		return code.size();
-	}
-	return env.codeSizeAt(addr);
+    assert(o_code != nullptr);  // Use get_code_size() to get code size.
```

Comment:
I forgot to change the argument to `byte const*`. But I have the follow up replacing getCode() just around the corner.